openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/spannerConnection.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/connections/database/spannerConnection.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "SpannerConnection",
+  "description": "Google Spanner Connection Config",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.services.connections.database.SpannerConnection",
+  "definitions": {
+    "spannerType": {
+      "description": "Service type.",
+      "type": "string",
+      "enum": [
+        "Spanner"
+      ],
+      "default": "Spanner"
+    },
+    "spannerScheme": {
+      "description": "SQLAlchemy driver scheme options.",
+      "type": "string",
+      "enum": [
+        "spanner+spanner"
+      ],
+      "default": "spanner+spanner"
+    }
+  },
+  "properties": {
+    "type": {
+      "title": "Service Type",
+      "description": "Service Type",
+      "$ref": "#/definitions/spannerType",
+      "default": "Spanner"
+    },
+    "scheme": {
+      "title": "Connection Scheme",
+      "description": "SQLAlchemy driver scheme options.",
+      "$ref": "#/definitions/spannerScheme",
+      "default": "spanner"
+    },
+    "hostPort": {
+      "title": "Host and Port",
+      "description": "Spanner APIs URL.",
+      "type": "string",
+      "default": "spanner.googleapis.com"
+    },
+    "credentials": {
+      "title": "GCS Credentials",
+      "description": "GCS Credentials",
+      "$ref": "../../../../security/credentials/gcsCredentials.json"
+    },
+    "connectionOptions": {
+      "title": "Connection Options",
+      "$ref": "../connectionBasicType.json#/definitions/connectionOptions"
+    },
+    "connectionArguments": {
+      "title": "Connection Arguments",
+      "$ref": "../connectionBasicType.json#/definitions/connectionArguments"
+    },
+    "instanceId": {
+      "title": "Instance ID",
+      "description": "Spanner Instance ID",
+      "type": "string"
+    },
+    "databaseId": {
+      "title": "Database ID",
+      "description": "Spanner Database ID",
+      "type": "string"
```

Comment:
we should allow comma separated values or an array of strings

Reply:
Alright. I will give it a try to improve.

Reply:
@harshach I have a question about how to handle multiple connection URLs to `create_generic_db_connection`. Moreover, I would like to know if we can make a connector have multiple inspectors. 

I would like to know:
1. How we can deal with multiple connection URLs in `get_connection`, and
2. How we can deal with multiple inspectors in a connection as we call `get_connection` in `set_inspector`.